.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-**Prep Release: (assigned to:)**
-_Due: <3-business-days-before-release>_
-- [ ] Merge all features and fixes into the `dev` branch
-- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
-- [ ] Update dependencies and increment extension version in the `dev` branch.
-- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
-- [ ] Close out or punt remaining GitHub issues for the current milestone
+**Prep DTFx Release: (assigned to:)**
+_Due: <2-3-business-days-before-release>_
+- [ ] Increment updated DTFx packages (either DT-AzureStorage only or DT-AzureStorage, DT-Core, and DT-Redis)
```

Comment:
So it's either just 
(A) AzureStorage or
(B) AzureStorage + Core + Redis all at once?

It might be good to clarify in the notes themselves, as well as how to determine which approach to take :)

Reply:
It should be AzureStorage, or AzureStorage, Core, Emulator, and Redis if Core was touched.

Reply:
Thanks! Good to know. I would still recommend to have it in the notes though, but no strong feelings about this

Reply:
This is just the increment step. Technically you increment Emulator at the same time as Core, so I just left it out to avoid confusion. I can update it if you have a preference and add "if changes were made to Core"

Reply:
@amdeel I would prefer the extra detail, yes :) Just to avoid confusion